src/muiComponents/Input/index.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { ReactElement } from "react"
+import { Box } from "@mui/material"
+
+interface Props {
```

Comment:
Is anyone against more descriptive prop naming? `TokenInputProps` @hammeiam @nkrulikovskiy

This can be problematic when there is a single file containing multiple components. Usually, you will see this pattern when there are smaller components.

Reply:
Yes, you are right.
This component is not finished on this PR. so, I deleted this component.